frame/contracts/src/weights.rs

```diff
@@ -152,1311 +153,1299 @@ pub trait WeightInfo {
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes((100 as Weight).saturating_mul(d as Weight)))
 	}
 	fn seal_random(r: u32, ) -> Weight {
-		(140_411_000 as Weight)
-			// Standard Error: 146_000
-			.saturating_add((566_687_000 as Weight).saturating_mul(r as Weight))
+		(124_927_000 as Weight)
+			// Standard Error: 407_000
+			.saturating_add((730_247_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn seal_deposit_event(r: u32, ) -> Weight {
-		(132_048_000 as Weight)
-			// Standard Error: 308_000
-			.saturating_add((818_622_000 as Weight).saturating_mul(r as Weight))
+		(135_014_000 as Weight)
+			// Standard Error: 892_000
+			.saturating_add((1_131_992_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn seal_deposit_event_per_topic_and_kb(t: u32, n: u32, ) -> Weight {
-		(1_080_578_000 as Weight)
-			// Standard Error: 2_337_000
-			.saturating_add((534_525_000 as Weight).saturating_mul(t as Weight))
-			// Standard Error: 460_000
-			.saturating_add((167_990_000 as Weight).saturating_mul(n as Weight))
+		(1_401_344_000 as Weight)
+			// Standard Error: 2_961_000
+			.saturating_add((701_918_000 as Weight).saturating_mul(t as Weight))
+			// Standard Error: 583_000
+			.saturating_add((169_206_000 as Weight).saturating_mul(n as Weight))
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().reads((100 as Weight).saturating_mul(t as Weight)))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes((100 as Weight).saturating_mul(t as Weight)))
 	}
 	fn seal_set_rent_allowance(r: u32, ) -> Weight {
-		(123_998_000 as Weight)
-			// Standard Error: 53_000
-			.saturating_add((155_113_000 as Weight).saturating_mul(r as Weight))
+		(146_753_000 as Weight)
+			// Standard Error: 117_000
+			.saturating_add((194_150_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn seal_debug_message(r: u32, ) -> Weight {
-		(120_514_000 as Weight)
-			// Standard Error: 93_000
-			.saturating_add((124_243_000 as Weight).saturating_mul(r as Weight))
+		(141_972_000 as Weight)
+			// Standard Error: 114_000
+			.saturating_add((164_981_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn seal_set_storage(r: u32, ) -> Weight {
-		(47_131_000 as Weight)
-			// Standard Error: 931_000
-			.saturating_add((4_033_062_000 as Weight).saturating_mul(r as Weight))
+		(549_424_000 as Weight)
```

Comment:
I have no idea how this was causes. But the value is not important. It is never used because it is the start-up time of the sandbox and not the cost of the actual call. The final value is calculated as  `seal_set_storage(1) - seal_set_storage(0)`.

Mind the error. It is pretty big. I think this is expected and OK.

Reply:
ah it is used like this, then yes it is definitely ok